packages/main/test/samples/Calendar.sample.html

```diff
@@ -0,0 +1,105 @@
+<header>
+	<h2 class="control-header">Calendar</h2>
+</header>
+
+<div class="component-package">@ui5/webcomponents</div>
+
+<div class="control-tag">&lt;ui5-calendar&gt;</div>
+
+<section>
+	<h3>Basic Calendar</h3>
+	<div class="snippet">
+		<div class="datepicker-width">
+			<ui5-calendar></ui5-calendar>
+		</div>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+		<ui5-calendar></ui5-calendar>
+	</xmp></pre>
+</section>
+
+<section>
+	<h3>Calendar with Minimum and Maximum Date - 7/7/2020 - 20/10/2020 format-pattern="dd/MM/yyyy"</h3>
+	<div class="snippet">
+		<div class="datepicker-width">
+			<ui5-calendar min-date="7/7/2020" max-date="20/10/2020" format-pattern="dd/MM/yyyy"></ui5-calendar>
+		</div>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+		<ui5-calendar min-date="7/7/2020" max-date="20/10/2020" format-pattern="dd/MM/yyyy"></ui5-calendar>
+	</xmp></pre>
+</section>
+
+<section>
+	<h3>Calendar with hidden week numbers</h3>
+	<div class="snippet">
+		<div class="datepicker-width">
+			<ui5-calendar hide-week-numbers></ui5-calendar>
+		</div>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+		<ui5-calendar hide-week-numbers></ui5-calendar>
+	</xmp></pre>
+</section>
+
+<section>
+	<h3>Calendar with multiple selection mode</h3>
```

Comment:
Mode or type? We have introduced it as type, so maybe better:
<h3>Calendar with selection type Multiple</h3>

Reply:
Done